cmd/cluster-init/update_sanitize_prow_jobs.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/dispatcher"
+)
+
+func updateSanitizeProwJobs(o options) error {
+	logrus.Println("Updating sanitize-prow-jobs config")
+	filename := filepath.Join(o.releaseRepo, "core-services", "sanitize-prow-jobs", "_config.yaml")
+	data, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return err
+	}
+	c := &dispatcher.Config{}
```

Comment:
Go-ism: Initialize intentionally empty values using `var`

```go
var c dispatcher.Config
```

Reply:
I tried this, but thought the `yaml#Unmarshal` was throwing an error. Turns out it must have been something else, so I will make this change after all.

Reply:
Outstanding